integration_tests/pages/overview/OverviewPage.ts

```diff
@@ -31,6 +31,16 @@ export default class OverviewPage extends Page {
     return this
   }
 
+  selectTab(targetTab: string) {
```

Comment:
This method on the Overview page class allows us to click tab headings to move from tab to tab 👍